package.json

```diff
@@ -279,23 +279,23 @@
     "@metamask/jazzicon": "^2.0.0",
     "@metamask/json-rpc-engine": "^10.0.0",
     "@metamask/json-rpc-middleware-stream": "^8.0.4",
-    "@metamask/keyring-api": "^17.6.0",
-    "@metamask/keyring-controller": "^22.0.0",
-    "@metamask/keyring-internal-api": "^6.1.0",
-    "@metamask/keyring-internal-snap-client": "^4.0.2",
-    "@metamask/keyring-snap-client": "^4.1.0",
+    "@metamask/keyring-api": "^18.0.0",
+    "@metamask/keyring-controller": "^22.0.1",
+    "@metamask/keyring-internal-api": "^6.2.0",
+    "@metamask/keyring-internal-snap-client": "^4.1.0",
+    "@metamask/keyring-snap-client": "^5.0.0",
     "@metamask/logging-controller": "^6.0.4",
     "@metamask/logo": "^4.0.0",
     "@metamask/message-manager": "^12.0.1",
     "@metamask/message-signing-snap": "1.1.2",
     "@metamask/metamask-eth-abis": "^3.1.1",
     "@metamask/multichain-api-client": "^0.6.0",
     "@metamask/multichain-api-middleware": "^0.1.1",
-    "@metamask/multichain-network-controller": "^0.5.1",
```

Comment:
Skipping 2 "majors" (controller not yet considered stable), but those versions are mostly peer dep bumps.